pkg/router/router.go

```diff
@@ -3,8 +3,7 @@ package router
 import (
 	"net/http"
 
-	"github.com/AsterNighT/software-engineering-backend/api"
-	//	_ "github.com/AsterNighT/software-engineering-backend/docs" // swagger doc
```

Comment:
No you should not delete `_ "github.com/AsterNighT/software-engineering-backend/docs" // swagger doc`, this is intended.